monkey/infection_monkey/monkey.py

```diff
@@ -239,10 +239,10 @@ def _subscribe_events(
         )
 
     @staticmethod
-    def _get_local_network_interfaces():
+    def _get_local_network_interfaces() -> List[IPv4Interface]:
         local_network_interfaces = get_local_network_interfaces()
         for i in local_network_interfaces:
-            logger.debug(f"Found local interface {i.address}{i.netmask}")
+            logger.debug(f"Found local interface {str(i.ip)}/{i.network.prefixlen}")
```

Comment:
Why is it still not `str(i)`?

Reply:
Because I missed it! 😆 Thanks for pointing it out!